src/components/portfolio/DetailsAnalyticsDebtEquityBar.tsx

```diff
@@ -24,24 +24,37 @@ const DetailsAnalyticsDebtEquityBar: React.FC<DetailsAnalyticsDebtEquityBarProps
   const series = [
     {
       name: t('portfolio.details.analytics.debtEquity'),
-      data: biggestTen.map((pos) => pos.stock.debtEquity),
+      data: biggestTen
+        .map((pos) => pos.stock.debtEquity)
+        .concat([portfolio.analytics.debtEquity]),
     },
   ];
 
   const options = {
+    tooltip: {
+      y: {
+        formatter: (value: number) => Math.round(value * 10000) / 10000,
```

Comment:
This could be moved to a helper function instead of duplicated, so it is easier to change.

Reply:
@an-p done ✅